site/soc/software/charts/osh/openstack-compute-kit/nova.yaml

```diff
@@ -63,6 +65,15 @@ metadata:
       dest:
         path: .values.pod.replicas.novncproxy
 data:
+  upgrade:
+    pre:
+      delete:
+        - type: pod
+          name: airship-nova-test
+          labels:
+            application: nova
+            component: test
+            release_group: airship-nova
```

Comment:
Looks fine though question is why we need to delete test pod only for nova . I don't see this change for neutron test.

Reply:
Seems like James already have upstream review for this and included in dev-patcher so we may no longer need this.
Upstream review: https://review.openstack.org/#/c/645218

Reply:
That review is clearer than this  :) 

Hard to understand without context. Thanks @kantarun !